ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2446,6 +2446,7 @@ periodics:
   - base_ref: master
     org: openshift
     repo: ci-tools
+    workdir: true
```

Comment:
This one is the first ref.
It is the workdir already.
Could you please try `ls` by a rehearse label?

Reply:
Sure thing, that seemed to work as expected. 

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/30963/rehearse-30963-periodic-prow-auto-aggregator-job-names/1554115010496565248

Reply:
Reverted back the `ls` change